### ISSUE TITLE
Day 1 Node JS pictures

### ISSUE THREAD

**Issue by dhack01** [Asabeneh/30-Days-Of-JavaScript#327]
would be great if the pictures could be updated. It looked pretty different when I installed

**Comment by devprotim** [comment]
Which ones exactly, can you point out?

**Comment by emmanuelmoon** [comment]
> Which ones exactly, can you point out?

I think he means that the version of Node has been updated.

**Comment by DevDario** [comment]
> > Which ones exactly, can you point out?
> 
> I think he means that the version of Node has been updated.

Yes, but they have to inform also which version (the most stable one) he as to download.
something similar to that happened to me, and let me say that a minimal mistake lake this can ruin everything 😅
